jparest/pom.xml

```diff
@@ -87,6 +87,16 @@
             </exclusions>
         </dependency>
 
```

Comment:
The version in this pom needs to change to 0.0.10 same as the one in the parent pom

Reply:
Where does the version get updated in this file? I can only see references to project.version?

Reply:
should be the one on line 7 
`<version>0.0.9</version>`

Reply:
Done :)